custom/conf/app.example.ini

```diff
@@ -694,6 +694,7 @@ ROUTER = console
 ;ENABLE_REVERSE_PROXY_AUTHENTICATION = false
 ;ENABLE_REVERSE_PROXY_AUTO_REGISTRATION = false
 ;ENABLE_REVERSE_PROXY_EMAIL = false
+;ENABLE_REVERSE_PROXY_NAME = false
```

Comment:
Use `ENABLE_REVERSE_PROXY_FULL_NAME` is better.

Reply:
sure, that works.  And presumably the same change throughout on other config options and go var names?   I can certainly do that.

Reply:
> sure, that works. And presumably the same change throughout on other config options and go var names? I can certainly do that.

I think so.

Reply:
The only thing I'd recommend leaving as just "name" is the default header `X-WEBAUTH-NAME`, since that seems to already be used in the wild.

Reply:
Done.